test/parallel/test-fs-access.js

```diff
@@ -63,62 +63,62 @@ assert.strictEqual(typeof fs.R_OK, 'number');
 assert.strictEqual(typeof fs.W_OK, 'number');
 assert.strictEqual(typeof fs.X_OK, 'number');
 
-fs.access(__filename, function(err) {
-  assert.strictEqual(err, null, 'error should not exist');
-});
+fs.access(__filename, common.mustCall((err) => {
+  assert.ifError(err);
+}));
 
-fs.access(__filename, fs.R_OK, function(err) {
-  assert.strictEqual(err, null, 'error should not exist');
-});
+fs.access(__filename, fs.R_OK, common.mustCall((err) => {
+  assert.ifError(err);
+}));
 
-fs.access(doesNotExist, function(err) {
+fs.access(doesNotExist, common.mustCall((err) => {
   assert.notEqual(err, null, 'error should exist');
   assert.strictEqual(err.code, 'ENOENT');
   assert.strictEqual(err.path, doesNotExist);
-});
+}));
 
-fs.access(readOnlyFile, fs.F_OK | fs.R_OK, function(err) {
-  assert.strictEqual(err, null, 'error should not exist');
-});
+fs.access(readOnlyFile, fs.F_OK | fs.R_OK, common.mustCall((err) => {
+  assert.ifError(err);
+}));
 
-fs.access(readOnlyFile, fs.W_OK, function(err) {
+fs.access(readOnlyFile, fs.W_OK, common.mustCall((err) => {
   if (hasWriteAccessForReadonlyFile) {
-    assert.equal(err, null, 'error should not exist');
+    assert.ifError(err);
   } else {
     assert.notEqual(err, null, 'error should exist');
     assert.strictEqual(err.path, readOnlyFile);
   }
-});
+}));
 
-assert.throws(function() {
-  fs.access(100, fs.F_OK, function(err) {});
+assert.throws(() => {
+  fs.access(100, fs.F_OK, (err) => {});
 }, /path must be a string or Buffer/);
 
-assert.throws(function() {
+assert.throws(() => {
   fs.access(__filename, fs.F_OK);
 }, /"callback" argument must be a function/);
 
-assert.throws(function() {
+assert.throws(() => {
   fs.access(__filename, fs.F_OK, {});
 }, /"callback" argument must be a function/);
 
-assert.doesNotThrow(function() {
+assert.doesNotThrow(() => {
   fs.accessSync(__filename);
 });
 
-assert.doesNotThrow(function() {
+assert.doesNotThrow(() => {
   var mode = fs.F_OK | fs.R_OK | fs.W_OK;
 
   fs.accessSync(readWriteFile, mode);
 });
 
-assert.throws(function() {
+assert.throws(() => {
   fs.accessSync(doesNotExist);
-}, function(err) {
+}, common.mustCall((err) => {
```

Comment:
Nit: it doesn't harm but I think the use of `common.mustCall()` is not necessary here.